demo/src/app/components/datepicker/demos/hebrew/datepicker-hebrew.html

```diff
@@ -3,9 +3,17 @@
   In Israel it is used for religious purposes and frequently as an official calendar for civil purposes.
 </p>
 
-<ngb-datepicker class="rtl" #dp [(ngModel)]="model" [displayMonths]="2" [firstDayOfWeek]="7">
+<ngb-datepicker class="hebrew rtl" #dp [(ngModel)]="model" [displayMonths]="1" [firstDayOfWeek]="7" [dayTemplate]="dt"
+[dayTemplateData]="dayTemplateData">
 </ngb-datepicker>
 
+<ng-template #dt let-date="date" let-data="data" let-selected="selected" let-currentMonth="currentMonth">
```

Comment:
This being a demo, shouldn't we use the short version `let-date` instead of `let-date="date"` ?

Reply:
Yep, thanks. Created this PR before $implicit was introduced. Will fix